entities/itemRepresentation.ts

```diff
@@ -3,14 +3,16 @@ import Item from "./item.js";
 export default class ItemRepresentation extends Phaser.GameObjects.Container {
     public item: Item;
     private quantityText: Phaser.GameObjects.Text;
+    private itemPriceText?: Phaser.GameObjects.Text
+    private priceTag?: Phaser.GameObjects.Image
+    private coinIcon?: Phaser.GameObjects.Image
 
     constructor(scene: Phaser.Scene, x: number, y: number, texture: string, frame: string | number, item: Item) {
         super(scene, x, y);
         this.item = item;
-
-        const image = scene.add.image(0, 0, item.sprite.texture, item.sprite.frame).setDisplaySize(64, 64);
+        const image = scene.add.image(0, 0, item.sprite.texture, item.sprite.frame).setDisplaySize(64, 64).setScale(1.3);
```

Comment:
I don't mind changing the size of the item's image, but then I think it should be consistent across the inventory - on the GIF attached when item is in transition between two slots it looks to have bigger size which results in weird animation.
Or is it just a gif?

Reply:
It was not on purpose, but I think it looks a bit cool! Like the item is "jumping" from slot x to slot y during the transition :D But I like your point about consistency so I will use the same size even during the transition